zmsadmin/tests/Zmsadmin/ProfileTest.php

```
@@ -8,176 +8,106 @@ class ProfileTest extends Base
 
     protected $parameters = [];
 
-    protected $classname = "Profile";
+    protected $classname = 'Profile';
 
-    public function testRendering()
+    public function testRendering(): void
```

PHPStan (2.2.7)

[error] Return type (void) of method BO\Zmsadmin\Tests\ProfileTest::testRendering() should be compatible with return type (Psr\Http\Message\ResponseInterface) of method BO\Slim\PhpUnit\Base::testRendering()


(method.childReturnType)